### DISCUSSION TITLE
Power Consumption Bed Isolation

### DISCUSSION THREAD

**Discussion by NuclearPhoenixx** [1coderookie/KobraGoNeoInsights#19] [Show and tell]
Maybe something for the docs, I isolated the bed yesterday with two layers of some (thermatape) PE tape that I stil had lying around here. In total it's a lot cheaper than most of the commercial sheets of isolation and without the weird aluminium foil on top that was originally intended to be used with home heaters. It's like a mm thicker too and I didn't need to change anything else on the bed since I could make cutouts for everywhere it was needed.

I also recorded the power consumption before and after for the bed only. It actually looks like about **~25% less power than stock** and obviously it heats up faster and holds heat for a LOT longer. I'll see how the glue 

![20231114_205330](https://github.com/1coderookie/KobraGoNeoInsights/assets/17517687/eec6b86c-eb4b-408e-be7a-fd6382be7287)

Feel free to use the image or something. The two layers are orthogonal to each other so I'm not worried about the small gaps.

**Comment by 1coderookie** [comment]
Thx :)  
I actually never measured the power consumption, but looking at how much better it keeps the heat and that at the same time the percentage being shown for heating when holding a specific bedtemp during a print, I was actually assuming it would be sth around 20-30% as well. Nice to have a number now, so I'll most likely add that. Also nice that you had some insulation laying around, I won't use the pic tho as I assume that most of the ppl don't have such a well equipped workshop like you probably do ;)  
I got 3 pieces of 300x300 insulation for ~8-9€ back then, so that's a good price imho as well. I also like the aluminum foil tbh, as it's supposed to reflect the heat and also gives a nice surface finish in the end, so I personally don't have a problem with that one. Only thing that kinda bugs me is that I have to use some Kapton tape being wrapped around the edges to make sure nothing will come off somehow.

**Reply by NuclearPhoenixx** [reply]
Well, I've read online that some users complain about the aluminium foil, because it can cut you on the edges, it's very delicate in general and most importantly it's at the wrong end of the insulation. It should be on the heated side, not on the cold side. After all it's designed to be installed on the wall behind your radiator. But hey, if it works, it's fine honestly.

Power consumption with the device at idle with only the heated bed on at 60°C went from ~64W to about 47W in comparison. I measured both 2 minutes after reaching the target temperature. In relative numbers that's pretty huge actually, but considering it's "only" 17W in absolute difference, it highly depends on how much you print if you buy it only for the power savings.

**Reply by 1coderookie** [reply]
>  most importantly it's at the wrong end of the insulation. It should be on the heated side, not on the cold side.  

Imho: no. The insulation takes up some heat and stores it. By doing so, it keeps the temps of the bed more stable and prevents cooling down of the bed to a certain degree. If you would just *reflect* the heat, you wouldn't have this 'storage' effect and the benefit of the insulation would be much smaller.  

And btw: those aluminum foil covered insulation pads (which are usually made of Styropor or Styrodur and not of this foam like the insulation pads for the beds here) for the wall behind your radiator are quite bad, as they don't allow any 'breathing' of the wall in that area (means, mold can became a real issue!). Yes, it's a 'hack' which kinda works (if at least applied properly) in terms of reflecting heat towards the room, but for the wall itself it's not very good at all. Better would be to take off the radiator, close that area using bricks and then re-mount the radiator again.. ;)   

I really am a 'fan' of insulating the heated bed. Not only cuz of the less power consumption (which really sums up throughout the year if you look at the price for a kW/h here in Germany..), but also cuz the bedtemp itself doesn't fluctuate that much anymore.  
I print a lot of PETG (with bedtemp 80°), and I think it helps to reduce the risk of warping off the buildplate as well. Besides that, if you leave the mobo and PSU underneath the printer where they originally are, it also helps to keep off some heat from those parts - especially when printing with higher bedtemps. And I assume (I didn't measure it tho as I always insulate the beds right away) it does have a positive effect on the temp inside the enclosures as well, as the heat mainly disappears from the upper side of the bed instead of radiating away towards the bottom right off that 'unprotected' heater coil of the PCB..

**Reply by NuclearPhoenixx** [reply]
>Imho: no. The insulation takes up some heat and stores it. By doing so, it keeps the temps of the bed more stable and prevents cooling down of the bed to a certain degree. If you would just reflect the heat, you wouldn't have this 'storage' effect and the benefit of the insulation would be much smaller.

Yes of course, I just meant that that the aluminium foil should be on the other side with the foam below it. That way it would reflect the thermal radiation from the heating coils and then insulate below that for thermal conduction with the foam. If the reflective foil is below the foam it's pretty worthless tbh, because there's only like 35°C worth of thermal radiation to reflect back into the foam instead of say 60°C. And because the foil is so thin, the location doesn't matter for the foam, so it'll be roughly the same temperature anyways due to thermal conduction. I think you get my point.

Most of the stuff you can buy online as "3D printer bed insulation" is just cut up radiator insulation sheets that are rebranded and resold at a gigantic markup. That's why it has the foil on the other side because that's where your radiator would normally be. Most of these sheets that I know of are exactly that type of PE foam, maybe just a bit thinner. You can treat yourself for one such thing by searching for a "Reflexionsfolie" on Amazon for example. There's loads of them for like 3€/sqm.
